docs/readmes/lte/dev_unit_testing.md

```diff
@@ -22,64 +22,33 @@ To SSH into the VM, run
 To run all existing unit tests, run
 
 ```bash
-[VM] cd magma/lte/gateway
-[VM] make test
+[VM] cd magma # or any subdirectory inside magma
+[VM] bazel test //...
 ```
 
-Note: Running all unit tests can take close to 15 minutes.
-
 ### Test Python AGW services
 
-To run only the Python unit tests, run
-
-```bash
-[VM] cd magma/lte/gateway
-[VM] make test_python
-```
-
-The list of services to test are configured in the following files.
-
-- `orc8r/gateway/python/defs.mk`
-- `lte/gateway/python/defs.mk`
-
-To run unit tests for a single Python service, select a name from the list of services and run
+To run only the Python unit tests, run the following command from inside the repository
 
 ```bash
-[VM] cd magma/lte/gateway
-[VM] make test_python_service MAGMA_SERVICE=<service_name>
-```
-
-In the case that unit tests for a single Python service are started multiple times, it is preferable to avoid the upstream installation process of the virtual environment by adding `DONT_BUILD_ENV=1` to the command, run
-
-```bash
-[VM] cd magma/lte/gateway
-[VM] make test_python_service MAGMA_SERVICE=<service_name> DONT_BUILD_ENV=1
+bazel test \$(bazel query \'kind(py_*, //...)\')
```

Comment:
this also collects manual tests (and other rules starting with py_ (that are not executed because they are not tests)), i.e., either
* search for py_test without label manual or
* bazel test on folders <- better readable